src/api/common/index.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
   server: require('./server'),
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
-  convertAmount: utils.convertAmount
+  unparseAmount: utils.unparseAmount,
```

Comment:
`toRippledAmount` or something ?  Mentioning only cause I had to read to see what `unparse` meant.